blockchain-scrapers/push.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+#set -x
+
+function usage {
+    echo "Usage: ./push.sh [-bdr] <service|all>"
+    echo "  -b      builds an image for a service (builds everything when run with 'all'"
+    echo "  -d      deploys a service in a stack with its name (deploys every service in a stack named 'all' when run with 'all'"
+    echo " 	-r 		removes a service (or every service if run with 'all')"
+    echo "  -h      display this usage guide"
+    exit 1
+}
+
+function deploy {
+	case $1 in
+		# deploy every service in a stack called 'all'
+		all) 		
+			find $blockchain_dir -name docker-compose-*.yml -exec docker stack deploy -c {} $1 \;
+			;;
+		# deploy a service in it's individual stack
+		btc|bch|ada|ltc|xmr|eth|erc20|xrp|usdt|trx|xlm|neo) 
+			find $blockchain_dir -name docker-compose-$1.yml -exec docker stack deploy -c {} $1 \;
+			;;
+		*)
+			echo "Error: unknown service '"$1"'"
+	esac
+}
+
+function build {
+	case $1 in
+		# build every service
+		all) 
+			find $blockchain_dir -name Dockerfile* -exec sh -c "docker build -f {} -t $DOCKER_HUB_LOGIN/blockchain-scrapers_$(sed -n -e 's/^.*-//p' {}) $GOPATH" \;
+			;;
+		# build a single service
+		btc|bch|ada|ltc|xmr|eth|erc20|xrp|usdt|trx|xlm|neo) 
```

Comment:
Would be better to have this dynamical so people don't need to modify the script when adding a blockchain

Reply:
Ok, I changed that and made a few other improvements as well.